frontend/packages/dev-console/src/components/monitoring/overview/MonitoringMetricsSection.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  Accordion,
+  AccordionItem,
+  AccordionToggle,
+  AccordionContent,
+  Grid,
+  GridItem,
+} from '@patternfly/react-core';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { requirePrometheus } from '@console/internal/components/graphs';
+import MonitoringDashboardGraph from '../dashboard/MonitoringDashboardGraph';
+import { workloadMetricQueries } from './queries';
+import './MonitoringSection.scss';
+
+const WorkloadGraphs = requirePrometheus(({ resource }) => {
+  const ns = resource.metadata.namespace;
+  const workloadName = resource.metadata.name;
+  const workloadType = resource.kind.toLowerCase();
+  return (
+    <>
+      <Grid className="co-m-pane__body">
```

Comment:
Why a grid ? Seems like it's just stacking on top of each other. Also don't wrap single elements in a fragment.